tcms/testruns/templates/testruns/get.html

```diff
@@ -240,8 +240,10 @@ <h5><span class="test-executions-count"></span> Results</h5>
                                                 <p class="test-execution-notes">
                                                     <strong>{% trans 'Notes' %}:</strong>
                                                 </p>
+                                                <strong>{% trans "Bugs and hyperlinks" %}: </strong>
+                                                [<a href="#" class="add-link-button" data-toggle="modal" data-target="#add-link-modal"> Add </a>]
```

Comment:
This should actually become an action item (menu at the right hand side of the row), similar to the menu items in TestPlan page. See the `refactor_testplan_ui` branch for how it is done. 1-click bug report also goes there, possibly file attachments too -> that would simplify the UI in the expandable area.